.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,29 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.188.0/containers/cpp
+{
+	"name": "C++",
+	"build": {
+		"dockerfile": "Dockerfile",
+		// Update 'VARIANT' to pick an Debian / Ubuntu OS version: debian-10, debian-9, ubuntu-20.04, ubuntu-18.04
+		"args": { "VARIANT": "ubuntu-20.04" }
+	},
+	"runArgs": [ "--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined"],
```

Comment:
@davidchisnall, any flags that would be interesting in here?

Reply:
Not really.  Docker's use of seccomp is placebo security at best, it's fine to leave this unconstrained.  On Windows this will be running in an isolated Linux VM, so there's no need to try to constrain anything further than that.